doozerlib/rpmcfg.py

```diff
@@ -159,9 +159,10 @@ def tito_setup(self):
             # one was specifically identified in the metadata, rename those
             # which we are not targeting. Tito/brew do not handle multiple .specs.
             if self.source.specfile is not Missing:
-                for f in os.listdir('.'):
-                    if os.path.isfile(f) and f.endswith('.spec') and f != self.source.specfile:
-                        self.logger.info('Renaming extraneous spec file before build: {}'.format(f))
+                for f in os.listdir(self.source_path):
+                    found_path = os.path.join(self.source_path, f)
+                    if os.path.isfile(found_path) and found_path.endswith('.spec') and found_path != self.specfile:
+                        self.logger.info('Renaming extraneous spec file before build: {} (only want {})'.format(found_path, self.specfile))
```

Comment:
output looks like ```
2019-10-30 18:36:16,776 INFO [rpms/openshift-kuryr] Renaming extraneous spec file before build: /tmp/working/sources/rpm_openshift-kuryr_kuryr-kubernetes/openshift-kuryr-kubernetes-rhel8.spec (only want /tmp/working/sources/rpm_openshift-kuryr_kuryr-kubernetes/openshift-kuryr-kubernetes.spec)
```
Think this could be tightened up a little?
```suggestion
                        self.logger.info('Renaming extraneous spec file before build: {} (only want {})'.format(f, self.source.specfile))
```